Крестики-нолики_1/PlayField.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+#include <cassert>
+#include <vector>
+
+class PlayField
+{
+public:
+	static constexpr int fieldSize = 3;
+	enum Cells
+	{
+		csEmpty,
+		csCross,
+		csNought
+	};
+	enum Status
+	{
+		fsInvalid,
+		fsNormal,
+		fsCrossesWin,
+		fsNoughtsWin,
+		fsDraw
+	};
+
+	class CellPos
+	{
+	public:
+		CellPos(int i, int j)
+		{
+			assert(i >= 0 || j >= 0 || i <= fieldSize - 1 || j <= fieldSize - 1);

```

Comment:
```suggestion
			assert(i >= 0 || j >= 0 || i < fieldSize || j < fieldSize );

```
И всё равно с условием не согласен.